sources/packages/backend/apps/api/src/services/disbursement-schedule/_tests_/unit/disbursement-schedule.getMaxTuitionRemittance.spec.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  NotificationActionsService,
+  SequenceControlService,
+} from "@sims/services";
+import { DisbursementScheduleService } from "../../disbursement-schedule-service";
+import { DataSource } from "typeorm";
+import { addDays } from "@sims/utilities";
+import { COE_WINDOW } from "../../../../utilities";
+import { COEApprovalPeriodStatus } from "../../disbursement-schedule.models";
+
+describe("DisbursementScheduleService-getCOEApprovalPeriodStatus", () => {
+  let service: DisbursementScheduleService;
+
+  beforeAll(async () => {
+    const dataSource = {} as DataSource;
+    dataSource.getRepository = jest.fn();
+    const sequenceService: SequenceControlService =
+      {} as SequenceControlService;
+    const notificationActionsService: NotificationActionsService =
+      {} as NotificationActionsService;
+    service = new DisbursementScheduleService(
+      dataSource,
+      sequenceService,
+      notificationActionsService,
+    );
+  });
+
+  it("Should throw error when disbursementDate is undefined", () => {
```

Comment:
Some test descriptions are using periods in the end and others are not. 
We can just have an agreement to care (or not care) for the test descriptions and follow it.

Reply:
ya. I also noticed that actually I added periods in my local (yet to commit), I liked it with periods as its that what we see in the summary 
![image](https://user-images.githubusercontent.com/77353155/222216331-50873a2c-c8c6-45c8-b52f-af090c4e43ef.png)

Reply:
@dheepak-aot @andrepestana-aot @sh16011993, can we agree on this?